### PR TITLE
feat(dotnetforce7): Add global error handler middleware

### PR DESCRIPTION
**Pull request by brukted**
This PR adds a global error handling middleware to the ASP.NET Web API project. The middleware catches unhandled exceptions and returns an appropriate error response. It improves application robustness and provides an opportunity centralized error logging and monitoring.

### CODE REVIEW

File: Backend/starter_project_dotnetforce7/BlogApp2/BlogApp.Identity/obj/BlogApp.Identity.csproj.nuget.dgspec.json
Comment (danielteferadeti) [inline Backend/starter_project_dotnetforce7/BlogApp2/BlogApp.Identity/obj/BlogApp.Identity.csproj.nuget.dgspec.json:1]: Files in this directory should be ignored

**Comment by brukted** [comment]
@danielteferadeti check it out

**Review by TigistShiferaw — APPROVED** [review]
Approved

**Review by TigistShiferaw — APPROVED** [review]
Approved

**Review by TigistShiferaw — APPROVED** [review]
Approved

**Pull request closed without merging**